src/SFA.DAS.ContentApi.Database/Tables/ContentType.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE [dbo].[ContentType]
```

Comment:
How will the lookup data be added to this table? - it would be a great opportunity to have this defined in the project from the start so that the service can be setup easily on a local database.

Reply:
Currently all we know is a 'banner' type. This can be seeded.

Reply:
This will be in the DB project so OK

Reply:
Done